package.json

```diff
@@ -49,42 +49,42 @@
   "scripts": {
     "precommit": "npm run --silent lint:staged",
     "commitmsg": "npm run --silent lint:commitmsg",
-    "clean": "./tooling/index.js exec -- rimraf dist .coverage",
+    "clean": "rimraf packages/node_modules/**/{.coverage,dist}",
     "bootstrap": "echo 'npm run bootstrap is no longer required. Hooray!'",
-    "prebuild": "./tooling/index.js exec -- rimraf dist",
-    "build": "./tooling/index.js build",
+    "prebuild": "npm run clean",
+    "build": "babel-node ./tooling/index.js build",
     "prebuild:docs": "rimraf ./docs/api",
     "build:docs": "documentation build --config documentation/config.yml --np --project-name ciscospark --project-version 1.x --format html --output ./docs/api --github ./packages/node_modules/ciscospark/src/index.js ./packages/node_modules/@ciscospark/plugin-phone/src/index.js ./packages/node_modules/@ciscospark/plugin-people/src/index.js",
     "check-karma-output": "./scripts/analyze-output.sh",
     "build:package": "echo 'deprecated; just use \"npm run --silent build\"'./tooling/build.js",
     "lint": "npm run --silent lint:js && npm run --silent lint:jenkinsfile && npm run --silent lint:package-names",
     "lint:commitmsg": "conventional-changelog-lint -e",
-    "lint:eslint": "eslint --ignore-pattern 'packages/node_modules/generator-ciscospark/generators/*/templates/**' --ignore-path .gitignore $(npm run --silent lint:eslint:ci-reporter)",
+    "lint:eslint": "eslint --ignore-pattern 'packages/node_modules/generator-ciscospark/generators/*/templates/**' --ignore-path .gitignore",
     "lint:eslint:ci-reporter": "bash -c '[[ -n \"${JENKINS}\" ]]' && echo \"-f checkstyle -o reports/style/eslint.xml\" || echo ''",
-    "lint:jenkinsfile": "grep -qc '.status' Jenkinsfile && echo 'Use result, not status, in Jenkinsfile' || true",
+    "lint:jenkinsfile": "node --eval \"fs.readFileSync('Jenkinsfile').includes('.status') ? console.log('Found \".status\" in Jenkinsfile. You meant \".result\"') && process.exit(1) : process.exit(0)\"",
```

Comment:
Might be getting close to time to move this into an actual script.

Reply:
yea. i'm working on getting [bowman](https://github.com/ianwremmel/bowman) finished so that we have a common package for monorepo tooling.